_includes/block/head.html

```diff
@@ -85,4 +85,7 @@
     font-display: swap;
   }
   </style>
+  {% if page.usemathjax %}
+  <script type="text/javascript" async src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
```

Comment:
@dalibor the mathjax docs don't use this cdn url (see: https://docs.mathjax.org/en/latest/web/start.html#using-mathjax-from-a-content-delivery-network-cdn). This CDN url is loading in mathjax 2.7.1 when latest is 3.x. Can we try their recommended jsdelivr urls? It's probably best if we at least peg to a major version e.g. their example jsdelivr url that specifies 3.x: https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js

Reply:
That works i.e. formulas are properly rendered:

<img width="1143" alt="Screen Shot 2022-09-19 at 16 55 28 PM" src="https://user-images.githubusercontent.com/48384/191047884-c782743d-4033-44a5-bd69-076e92f45707.png">